perllib/FixMyStreet/Reporting.pm

```diff
@@ -123,6 +123,8 @@ sub construct_rs_filter {
         $where{"$table_name.state"} = [ FixMyStreet::DB::Result::Problem->fixed_states() ];
     } elsif ( $self->state ) {
         $where{"$table_name.state"} = $self->state;
+    } elsif ($self->cobrand->call_hook('dashboard_export_include_all_states')) {
+        # Do nothing, want all states
```

Comment:
Might be good to have a test that this works

Reply:
It did not. :) Due to the confirmed date range part of the query, which I've changed to created - probably better for waste anyway, given the DD reports that sit unconfirmed for weeks.